definitions/test/policy_test_HuisartsVVTInzage.json

```diff
@@ -0,0 +1,134 @@
+{
+  "test:HuisartsVVTInzage": {
+    "organization": {
+      "format": {
+        "ldp_vc": {
+          "proof_type": [
+            "JsonWebSignature2020"
+          ]
+        },
+        "jwt_vc": {
+          "alg": [
+            "PS256"
+          ]
+        },
+        "ldp_vp": {
+          "proof_type": [
+            "JsonWebSignature2020"
+          ]
+        },
+        "jwt_vp": {
+          "alg": [
+            "ES256"
+          ]
+        }
+      },
+      "id": "pd_any_care_organization",
+      "name": "Care organization",
+      "purpose": "Finding a care organization",
+      "input_descriptors": [
+        {
+          "id": "id_uzicert_uracredential",

```

Comment:
Same question regarding accepted DID methods. Preferably limit to subject to `did:web` prefix, but needs to be discussed first